libs/template/presets/src/fulfillment/offline.ts

```diff
@@ -67,7 +66,7 @@ export function offlineServiceWorkerFulfillmentFeatures(
     new IndexedDbFeature(config?.indexedDb),
     new RouterFeature(),
     new (class extends BapiAuthFeature {
```

Comment:
```suggestion
    new BapiAuthFeature()
```

Reply:
But we need to drop `BapiAuthPlugin` for SW or I missed something?

Reply:
Ah yes, let's then have this in a separate class/file, you can call it `SwBapiAuthPlugin` which removed the plugins.